Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "tree-sitter-gleam"
 description = "gleam grammar for the tree-sitter parsing library"
-version = "0.26.2"
+version = "0.27.0"
```

Comment:
Don't update the package version; I'll do that once all changes for v0.27.0 are in :grin: